### ISSUE TITLE
Error: Cannot find module 'rimraf'

### ISSUE THREAD

**Issue by tafelito** [Urigo/meteor-client-bundler#67]
Installing latest MCB throws this error about the `rimraf` dependency. I saw this was added later, not sure why it might not being installed as it appears as a dependency. I tried to install it manually but is still failing

Any ideas?

**Comment by ardatan** [comment]
Could you update `meteor-client-bundler` to `0.5.8`

**Comment by tafelito** [comment]
the error is gone but the bundle still not working. I created a new react app using create-react-app and then a new meteor app with the --bare option. The first error happens when running the bundler. It says `While processing files with static-html (for target web.browser.legacy):
client/main.html:12: Expected <head> or <body> tag`. I solved that by removing the static-html package from the meteor app.  
Then I run the bundler again and start the app but it fails to compile. It throws `Module not found: Can't resolve 'meteor/meteor'`

This is how my index.js looks like

```js
import "meteor-client";
import React from "react";
import ReactDOM from "react-dom";
import { Meteor } from "meteor/meteor";

import "./index.css";
import App from "./App";
import registerServiceWorker from "./registerServiceWorker";

Meteor.startup(() => {
  ReactDOM.render(<App />, document.getElementById("root"));
  registerServiceWorker();
});

```

**Comment by ardatan** [comment]
Just remove `static-html` from `.meteor/packages`.

**Comment by tafelito** [comment]
Yes, as I put on my comment I had to do that in order to start the meteor app. But then the react app is the one failing now

**Comment by ardatan** [comment]
Then, you have to use custom bundler configuration instead of using `-s` parameter.

**Comment by tafelito** [comment]
this is how I run the bundler `meteor-client bundle -s api --url http://localhost:8000` and the meteor-client.js is being genereated under node_modules

**Comment by tafelito** [comment]
Ok I created a config file and now it seems to be working. I copy the default config from the bundler. Not sure what is the difference tho. 

Thanks

**Comment by tafelito** [comment]
There is actually one more thing. I had to add the react-meteor-data on the imports section of the config file and also react & react-dom as an external npm package but I'm still getting a warning that react-meteor-data requires react as a peerDependency.

Also when trying to use Meteor.user() it throws this error `Uncaught TypeError: __WEBPACK_IMPORTED_MODULE_1_meteor_meteor__.Meteor.user is not a function`

Do you want me to open a new issue?

**Comment by tafelito** [comment]
Yes I did that, but I still get the warning

**Comment by tafelito** [comment]
I'm also trying to add any of this packages `blaze-html-templates gadicc:blaze-react-component` so I can load Blaze templates from react but the first one is totally being ignored even when generating the tmp meteor project before bundling it and the other one it does seems to get it bundled but when I try to import it it says module not found

**Comment by ardatan** [comment]
Please do not add any html package to meteor client configuration.

**Comment by tafelito** [comment]
so how do I load the Blaze template? 
I'm not able to load the accounts-ui package templates nor the accounts-password
I just want to run the todo app demo that uses the accounts-ui package
